api.md

```diff
@@ -878,6 +884,216 @@ SELECT reorder_chunk('_timescaledb_internal._hyper_1_10_chunk', 'conditions_devi
 
 runs a reorder on the `_timescaledb_internal._hyper_1_10_chunk` chunk using the `conditions_device_id_time_idx` index.
 
+---
+## Compression :community_function: [](compression)
+TimescaleDB allows users to 
+- configure their hypertables for compression 
+- compress chunks of data manually or based 
+on policy.  
+
+Policy based compression will allow the user to compress a chunk when it has reached the specified age.
+The user may also manually compress a chunk or sets of chunks.  For more detailed discussion of this function <insert 
+tag to tutorial>.
+
+>:WARNING: Compression is only available for Postgres 10.9+
+ or 11 in order to use this feature. 
+ 
+ #### Restrictions
+ Version 1.5 does not support altering or inserting data into compressed chunks.  The data can be queried without any 
+ modifications, however if you need to backfill or update data in a compressed chunk you will need to decompress the chunk(s)
+ first.
+
+
+*	[alter table (compression)] (#compression_alter-table))
+*	[compress Table SEGMENT BY] (#compress_table-segmentby)
+*	[compress Chunk](#compress_chunk)
+*	[compress Policy] (#compress_policy)
+*	[compress Policy Remove] (#compress_policy_remove)
+*	[decompress Chunk] (#decompress_chunk)
+
+## ALTER TABLE (Compression) :community_function: [](compression_alter-table)
+
+'ALTER TABLE' statement is used to set table data organization characteristics for a hypertable 
+to support compression.
+
+#### Parameters
+|Name|Description|
+|---|---|
+| `table_name` | Name of the hypertable that will support compression |
+| `column_name` | name of the column used to order by and/or segment by|
+
+#### with cluase options [](compression_alter-table)
+|Name|Description|
+|---|---|
+| `timescale.compress_orderby` | Use of order by to organize data within hypertable |
+| `timescaledb.compress_segmentby` | Use of segment by to organize data within hypertable|
+
+The syntax (ORDER BY ONLY) is:
+
+``` sql
+ALTER TABLE <table_name> SET (timescale.compress,timescale.compress_orderby
+= '<column_name>, <column_name>');
+```
+#### Sample Usage [](compression_alter-table)
+Set Hypertable to use ORDER BY option for data organization
+
+```sql
+ALTER TABLE metrics SET (timescaledb.compress, timescaledb.compress_orderby = 'device_id, time');
+```
+
+The syntax (ORDER BY and SEGMENT BY) is:
+
+``` sql
+ALTER TABLE <table_name> SET (timescaledb.compress, timescaledb.compress_orderby = 
+<'column'>, timescaledb.compress_segmentby = <'column'>);
+```
+#### Sample Usage [](compression_alter-table)
```

Comment:
The two sample usage sections should be combined under one header

Reply:
done